credentialz/README.md

```diff
@@ -222,8 +222,10 @@ stream := RotateHostCredentials()
 stream.Send(
     RotateHostCredentialsRequest {
         server_keys: ServerKeysRequest {
-            public_key: "A....=",
-            private_key: "A....=",
+            auth_artifacts: []AuthenticationArtifacts{
+                private_key: []bytes("...."),
```

Comment:
Do you want to specify in here the format of this? I.e. base64 encoded openssh keys and openssh ssh certificates?